### ISSUE TITLE
azure-themes DetailsList focus indicator doesn't meet contrast ratio minimum

### ISSUE THREAD

**Issue by raharri** [microsoft/fluentui#26850]
### Environment Information

- **Package version(s)**: azure-themes 8.5.58

### Describe the issue:

The focus indicator for a DetailsList row when the DetailsList has selectionMode={SelectionMode.none} does not meet minimum contrast ratio for non-text contrast

#### Actual behavior:

![image](https://user-images.githubusercontent.com/57734445/218875336-e348bb8b-b458-464f-b470-d1aaba6caaee.png)

#### Expected behavior:

The non-themed DetailsList focus indicator is looks like it's just black, I think that could work?

### If applicable, please provide a codepen repro:

https://codepen.io/raharri/pen/qBmMxdj

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: Azure IoT Hub (and likely other Azure extensions)

**Comment by kaimihata** [comment]
Is this a bug with the azure-themes package? And if so, is there an eta on a fix?

**Comment by smhigley** [comment]
Yes, this should be owned by the azure-themes folks -- the focus ring color is coming from there. Just updated the assignment :)

**Comment by Jacqueline-ms** [comment]
Just saw this - this has since been resolved for both 7.0 and 8.0. Closing this issue :)
https://github.com/microsoft/fluentui/pull/28966
https://github.com/microsoft/fluentui/pull/28781
